### ISSUE TITLE
Fixed Nonce implementation?

### ISSUE THREAD

**Issue by NikoCosmico01** [nfc-tools/miLazyCracker#38]
Hi,
I would like to know if there is a way to make miLazy working even with a Mifare card with a fixed nonce, also known as prng fixed. I've been trying so hard to make it work but, even changing some parameters (es. increasing probe size) it doesn't work. I'm actually using a pn532 via Kali.
The card I am trying to retrieve the keys from is seen as a Mifare Classic 1k but I suspect it being a Mifare Plus (which is compatible with the Classic 1k one).
Hope you can help.

**Comment by Hmvgit** [comment]
Plus one on this.
I think it could also be the Fudan card or mifare plus 1k
As nonces are definitely static.
I hope the tech guys can use the commit https://github.com/RfidResearchGroup/proxmark3/commit/b37a4c14eb497b431f7443b9f685d7f2e222bfa0 and make this working.

**Comment by webmagic86** [comment]
@NikoCosmico01 @Hmvgit any update for static nonces?

**Comment by NikoCosmico01** [comment]
> @NikoCosmico01 @Hmvgit any update for static nonces?

No, at the end I bought the Proxmark3 Easy for less than 60 Euros.

**Comment by webmagic86** [comment]
> No, at the end I bought the Proxmark3 Easy for less than 60 Euros.

Was PM3 able to recover the keys from your card? Ciao.

**Comment by NikoCosmico01** [comment]
> > No, at the end I bought the Proxmark3 Easy for less than 60 Euros.
> 
> Was PM3 able to recover the keys from your card? Ciao.

Yes, using Iceman firmware.

**Comment by AndreaCovelli** [comment]
> > No, at the end I bought the Proxmark3 Easy for less than 60 Euros.
> 
> Was PM3 able to recover the keys from your card? Ciao.
Ciao @webmagic86, i saw that in an issue on proxmark3 repository you were able to retrieve keys from a fudan card with a pm3: how did you do that? Which commands did you use? Many thanks.
